app/models/user.rb

```diff
@@ -6,7 +6,7 @@ class User < ApplicationRecord
   validates :name, presence: true, uniqueness: true
   validates :email, uniqueness: true, allow_blank: true, format: {
     with: EMAIL_REGEX,
-    message: 'has invalid format',
+    message: I18n.t('.invalid')
```

Comment:
we need to do it like
`message: :has_invalid_format`

same applies to other model translations as well in which we are adding an error
we can discuss if you have any doubt in this